onlyOffice/src/test/java/org/exoplatform/platform/qa/ui/onlyOffice/OnlyOfficeTestIT.java

```diff
@@ -0,0 +1,137 @@
+import org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.*;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceSettingManagement;
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.exoplatform.platform.qa.ui.selenium.TestBase;
+import org.openqa.selenium.Keys;
+
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.*;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_COMPOSER_SHARE_BUTTON;
+import static org.exoplatform.platform.qa.ui.selenium.locator.HomePageLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.locator.HomePageLocator.ELEMENT_BAR_PROGRESS;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_CLOSE_DOCUMENT_PREVIEW1;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.*;
+
+
+public class OnlyOfficeTestIT extends Base {
+
+  HomePagePlatform            homePagePlatform;
+  NavigationToolbar           navigationToolbar;
+  SiteExplorerHome            siteExplorerHome;
+  ManageLogInOut              manageLogInOut;
+  ConnectionsManagement       connectionsManagement;
+  ActivityStream              activityStream;
+  SpaceManagement             spaceManagement;
+  TestBase                    testBase;
+
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+
+    homePagePlatform            =new HomePagePlatform(this);
+    navigationToolbar           =new NavigationToolbar(this);
+    siteExplorerHome =new SiteExplorerHome(this);
+    manageLogInOut = new ManageLogInOut(this);
+    //manageLogInOut.signInCas(PLFData.DATA_USER1, "gtngtn");
+    spaceManagement             =new SpaceManagement(this);
+
+    activityStream      =new ActivityStream(this);
+    manageLogInOut.signInCas("root", "gtn");
+    connectionsManagement = new ConnectionsManagement(this);
+    if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+      $(ELEMENT_SKIP_BUTTON).click();
+    }
+  }
+
+  @Test
+  public void EditOnline_BTN_US01 () {
+    String Document = "OO_test";
+    info("Upload document");
+    homePagePlatform.goToDocuments();
+    siteExplorerHome.uploadFile(Document + ".docx");
+    info("Open document");
+    siteExplorerHome.checkButtonDocument(Document);
+    siteExplorerHome.clickViewDocumentButton();
+    switchTo().window(1);
+    siteExplorerHome.clickEditOnine();
+    switchTo().window(2);
+    switchTo().frame(0);
+    $(byText(Document + ".docx")).waitUntil(Condition.visible,Configuration.openBrowserTimeoutMs);
```

Comment:
We should add more assertions to make sure the document is correctly open and contains the right content.